conan/api/model/list.py

```diff
@@ -277,6 +278,7 @@ def refs(self):
 
     @staticmethod
     def prefs(ref, recipe_bundle):
+        """ Get all the package references for a given recipe reference given a bundle."""
```

Comment:
I think we need to improve the "bundle" naming, it was there from a very early iteration, and it is not very clear.
We might even want to improve the interface, methods names?